### ISSUE TITLE
How can we make sure that all GMs are aware of the content created by other GMs

### ISSUE THREAD

**Issue by crimsonknave** [our-stories/west-marches#45]
I expect we'll run into a some issues where there's some lore or an NPC that was introduced in one session that gets overwritten in another session by another GM. For example, in one session we meet the captain of the guard and in another we meet a *different* captain of the guard.

The wiki is an easy place for us to store information, but it's also an easy place for that stuff to get lost. There's no notifications on things being updated and it's up to everyone who's contributing content to link things so they're discoverable. If we stored things in the repo we could enforce PRs and then everyone would get a notification. Additionally, there'd be a chance for folks to chime in and note that there's a similar NPC, or that this would connect really well to this other thing.

There are doubtless lots of ways we can address this. I don't have one in mind, I just think it's something we should be thinking about.

**Comment by djigneo** [comment]
Couple of thoughts off the top of my head. Not necessarily correct answers, just possible solutions:

- Single place for session recaps / summaries so GMs can at least read through them to get a sense of what has happened / what NPCs were introduced etc.
- Don't care about inconsistency - so what if there are two captain of the guard, etc.
- GMs only touch stuff they create as much as possible.
- Tool / compendium that allows for quick searching so it's easy for GMs / players to look up known information.

**Comment by zmoazeni** [comment]
Thanks @crimsonknave for bringing this up! I **want GMs and Players** to change Waldenstone. The surrounding areas. form bonds with the NPCs, etc. I **want the region to come _alive._**

In the short term, I am closer to @djigneo's line of thinking here. I don't know if we _need_ to care _that much_ about consistency up front.

One lesson to learn as a GM is to roll with the punches within the game and build on whatever your Players. If one GM changes things within the town I totally get not wanting another GM to fully step on their toes. The mechanism in my mind is for the Player to speak up, and the GM to adjust accordingly within the game.

I would like to see big events recorded in the repository somewhere. I'm leaning more towards commits rather than wiki because it's easier for GitHub Pages, and easier to organize. But I don't have a strong preference there. Other than that, I'd like to see us run into this problem first and see what the appropriate steps are to prevent it again. Because it's possible we won't run into it, or it will take care of itself.

**Comment by crimsonknave** [comment]
To be clear, I don't think it's a problem we'll have soon. But I wanted to point it out so that we're thinking about it ahead of time and can build in some guard rails around it. I'd love to see the setting get built out in pages instead of the wiki. I think that would solve most of my issues (assuming we enforce PRs so there's a trail of what's changed that's easier to find that looking at the commit history).

**Comment by zmoazeni** [comment]
Yeah I am in favor of the PR route! Perhaps we setup CODEOWNERS or a review policy where another GM has to approve the setting changes.

That could be another route to help avoid unhappy gaming scenarios.
